actors/builtin/market/market_state.go

```diff
@@ -100,7 +100,9 @@ func (st *State) updatePendingDealState(rt Runtime, state *DealState, deal *Deal
 		// unlock client collateral and locked storage fee
 		clientCollateral := deal.ClientCollateral
 		paymentRemaining := dealGetPaymentRemaining(deal, state.SlashEpoch)
-		st.unlockBalance(lt, deal.Client, big.Add(clientCollateral, paymentRemaining))
+		if err := st.unlockBalance(lt, deal.Client, big.Add(clientCollateral, paymentRemaining)); err != nil {
+			rt.Abortf(exitcode.ErrIllegalState, "failure unlocking client balance: %s", err)
```

Comment:
@anorth please check that I am using the correct exitcode for the new aborts.  I'm using ErrIllegalState everywhere but don't know the system well enough to be sure if this is correct.

Reply:
FYI `builtin.RequireNoErr` gives a more concise check, usually.